CHANGES.md

```diff
@@ -4,16 +4,20 @@ This file is intended for use by the developers of this repository and of the re
 
 ## July 2024
 
-**WBS Deploy 3.0.0** comes with the following WBS Images:
-
-| WBS Image             | WBS Version | Upstream Version     | Base Image                    | Distribution    |
-| --------------------- | ----------- | -------------------- | ----------------------------- | --------------- |
-| Wikibase Image        | 3.0.0       | MediaWiki 1.42.1     | PHP 8.3.8 apache              | Debian Bookworm |
-| WDQS Image            | 1.0.0       | WDQS 0.3.142         | JRE Eclipse Temurin 8u412-b08 | Debian Bookworm |
-| WDQS Frontend Image   | 1.0.0       | N/A                  | Nginx 1.27.0                  | Debian Bookworm |
-| WDQS Proxy Image      | 1.0.0       | N/A                  | Nginx 1.27.0                  | Debian Bookworm |
-| QuickStatements Image | 1.0.0       | N/A                  | PHP 8.3.8 Apache              | Debian Bookworm |
-| ElasticSearch Image   | 1.0.0       | ElasticSearch 7.10.2 | N/A                           | N/A             |
+### WBS Deploy 3.0.0
```

Comment:
Let's title this simply "WBS 3.0.0" according to `variables.env`, and add a line for WBS Deploy version, like:

"comes with the following WBS releases:

WBS Deploy 3.0.0
...and WBS Images...
"

Reply:
looking at this later, thanks for the input.